### ISSUE TITLE
Image.Display()

### ISSUE THREAD

**Issue by plaresmedima** [QIB-Sheffield/WEASEL#47]
@JSousa-UoL 

in the Image class can you add this method

    def Display(self):
        self.DisplayImage()

and in the Series class add this

    def Display(self):
        self.DisplaySeries()

I know its duplication for now but we can harmonize later. With this definition I can do

Image.Display()
ImageList.Display()
Series.Display()
SeriesList.Display()

and later we can add

Study.Display()
etc..

Basically, Display() creates the appropriate display whatever the object is.

**Comment by JSousa-UoL** [comment]
Ok, I'm sure it's ok to just rename DisplayImage() and DisplaySeries() to Display() as they belong to different classes. Sounds right to you?

**Comment by plaresmedima** [comment]
yes except you are breaking backwards compatibility if you just replace
them. Are you using DisplayImage() and DisplaySeries() in other places?

Prof. Steven Sourbron, PhD
Chair in Medical Imaging Physics
University of Sheffield, UK
*https
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>*
*Sheffield abdominal imaging research page*
<https://www.sheffield.ac.uk/medicine/research/research-themes/medical-imaging/medical-imaging-research/abdominal-imaging>
*Open source initiative in perfusion imaging* <http://www.osipi.org>
*Magnetic resonance imaging biomarkers for chronic kidney disease
<https://renalmri.org/>*

**Comment by JSousa-UoL** [comment]
It shouldn't be a big problem, DisplayImage() and DisplaySeries() are only used in the Pipelines scripts as Gaussian, Square, Threshold... They can be easily changed since they are wrapper functions themselves.

**Comment by plaresmedima** [comment]
Let's move them to a "retired" list first - once you have other users
developing pipelines you loose control over what is used where. Kanishka
may be using them for instance and you wouldn't know. Once we are sure they
are not used anywhere then the retired list can be cleared.

Prof. Steven Sourbron, PhD
Chair in Medical Imaging Physics
University of Sheffield, UK
*https
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>*
*Sheffield abdominal imaging research page*
<https://www.sheffield.ac.uk/medicine/research/research-themes/medical-imaging/medical-imaging-research/abdominal-imaging>
*Open source initiative in perfusion imaging* <http://www.osipi.org>
*Magnetic resonance imaging biomarkers for chronic kidney disease
<https://renalmri.org/>*

**Comment by JSousa-UoL** [comment]
I was just wondering where to put some of those "old" scripts. I'll create a folder called "Retired" in "Pipelines" folder and remove "Steven-Test" folder.

**Comment by JSousa-UoL** [comment]
Hi @plaresmedima , please have a look at this solution when possible. You may now replace in your files DisplayImage() and DisplaySeries() by Display() only. You still need to use DisplayImages() for ImageList as there's no class for Images in Developer Tools

**Comment by plaresmedima** [comment]
That's working I think Joao, thanks

Prof. Steven Sourbron, PhD
Chair in Medical Imaging Physics
University of Sheffield, UK
*https
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron
<https://www.sheffield.ac.uk/medicine/people/iicd/steven-sourbron>*
*Sheffield abdominal imaging research page*
<https://www.sheffield.ac.uk/medicine/research/research-themes/medical-imaging/medical-imaging-research/abdominal-imaging>
*Open source initiative in perfusion imaging* <http://www.osipi.org>
*Magnetic resonance imaging biomarkers for chronic kidney disease
<https://renalmri.org/>*
